planning/autoware_path_generator/include/autoware/path_generator/node.hpp

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 TIER IV, Inc.
```

Comment:
Can you put the header files to the `src` directory since they will not be included by other packages like the mission planner?
https://github.com/autowarefoundation/autoware.universe/tree/main/planning/autoware_mission_planner/src/mission_planner